roonapi/roonapi.py

```diff
@@ -444,11 +527,12 @@ def play_media(self, zone_or_output_id, path, action=None):
 
                 load_opts["offset"] += PAGE_SIZE
             if searched >= total_count and found is None:
-                LOGGER.error(
```

Comment:
I need to log an error for home assistant users trying to debug their searches (otherwise it's really hard to use) - but perhaps we could add an optional parameter to`play_media` - perhaps `report_error=True` and you can then call with `False`?

Reply:
Thanks for the prompt review. I agree with your suggestions and have committed and pushed changes to address them. The only non-trivial change is this one, adding an optional parameter to play_media for logging an error. My tests today were all successful so I believe my change here is ok but I wanted to get your review and perform some additional testing before I mark this as resolved. Let me know what you think.